pydicom/config.py

```diff
@@ -71,6 +72,41 @@ def DS_decimal(use_Decimal_boolean=True):
 logger.addHandler(handler)
 
 
+image_handlers = []
+
```

Comment:
One last small thing... I think we should have a doc string for `image_handlers`, so it can show up in reference documentation.  My [sample code](https://github.com/pydicom/pydicom/issues/18#issuecomment-314960744) way back in the discussion had a doc string that could be a starting point, although I think it is too long.  The main point is that the user can control the order that handlers (or which handlers) are called, if they wish, by changing the list.